tests/testthat/_snaps/windows/report.htest-kruskal.md

```diff
@@ -0,0 +1,12 @@
+# report.htest-kruskal report
+
+    Code
+      report(x, verbose = FALSE)
+    Output
+      Effect sizes were labelled following Field's (2013) recommendations.
+      
+      The Kruskal-Wallis rank sum test testing the difference in ranks between
+      airquality$Ozone and as.factor(airquality$Month) suggests that the effect is
```

Comment:
Should we be cleaning up names here? That is, should this be `airquality$Month` instead of `as.factor(airquality$Month)`? I think insight has some helpers that can do this.

If you agree, you can create a new issue to track this.

Reply:
Thanks, yes good idea, I will create a new issue so this can be done systematically!

Reply:
#401